Library/Homebrew/upgrade.rb

```diff
@@ -134,10 +134,10 @@ def check_broken_dependents(installed_formulae)
       end
     end
 
-    def check_installed_dependents(args:)
+    def check_installed_dependents(f, args:)
```

Comment:
this shadows another local variable `f`. how about calling it `installed_formulae`?